### DISCUSSION TITLE
名畫《吶喊》裡面的人並沒有在吶喊

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1554] [General]
你一定看過一幅名叫《吶喊》的世界名畫，但幾乎所有人都會產生的誤會是
真正在吶喊的，其實不是那位畫裡的人物
_ 
而是畫中的背景
_ 
據畫家孟克 本人解釋這幅畫的緣由
是源自一次外出散步時
他忽然沒由來地感到一陣焦慮
_ 
就在停下腳步並試圖平息這股焦慮時
他注意到眼前那片被夕陽染成血色的天空
並感受到大自然那由四面八方撲面襲來
劇烈，又仿佛永無止盡的刺耳 #吶喊
_ 
因此這件作品真正的核心
是透過線條、顏色與構圖等編排
將來自大自然的吶喊，以視覺方式加以呈現
而畫中人物那看似也在吶喊的動作
其實只是面對吶喊時，所表現出的驚慌失措

**Comment by zhiron544** [comment]
我覺得這篇文章很有趣，讓我想起了我第一次看到這幅畫時的感覺。我當時也被畫中的人物和背景所吸引，但沒有想到他們之間的關係。我也想知道畫家是在什麼情況下創作了這幅畫，他是否經歷了什麼特別的事情。
